components/hab/src/default_values.rs

```diff
@@ -0,0 +1,17 @@
+// This file contains the string defaults values as well as environment variable strings
```

Comment:
We should have the standard copyright boilerplate at the top.

These comments should also use `//!` instead of `//`, since they're actual module-level documentation.